### PR TITLE
Fixes defects in the PDF report if a long device name is specified

### PR DESCRIPTION
**Pull request by jboddey**
See attached PDF

[2024-04-09T13_09_48.pdf](https://github.com/google/testrun/files/14918718/2024-04-09T13_09_48.pdf)

### CODE REVIEW

File: framework/python/src/common/testreport.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 DATE_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 RESOURCES_DIR = 'resources/report'
-TESTS_FIRST_PAGE = 12
+TESTS_FIRST_PAGE = 9
```
Comment (jhughesbiot) [inline framework/python/src/common/testreport.py:26]: This should be dynamic depending on the size of the header.  Making it 9 leaves a lot of dead space with any manufacturer not having too long of a name problem.

**Review by jhughesbiot — COMMENTED** [review]
Spacing for longer names cuts off multi-page module reports.  Needs to dynamically account for this extra space needed for the header.

image

**Review by KrystsinaT — COMMENTED** [review]
Verified. The Post request to create a device with more than 28 characters results in 400 {"error":"Invalid request received"}.

File: framework/python/src/common/testreport.py (hunk 1 above)
Comment (jboddey) [reply framework/python/src/common/testreport.py]: It has now been set to 11 which should be enough space for a device with a long name, without causing much white space for short names.

**Comment by jhughesbiot** [comment]
> Spacing for longer names cuts off multi-page module reports. Needs to dynamically account for this extra space needed for the header.
> 
> image

Updated the row size to account for the updated CSS changes so this should be resolved now.

**Review by jhughesbiot — APPROVED** [review]

**Pull request merged**